tutorials/generate-events.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: Generating Events
+---
```

Comment:
I think you're right about this. I will add a section at the beginning of each tutorial concisely setting the context.